modules/upi-provisioning-new-hosts-in-a-upi-cluster.adoc

```diff
@@ -9,7 +9,7 @@ You can use the Bare Metal Operator (BMO) to provision bare-metal hosts in a use
 
 [NOTE]
 ====
-To provision bare-metal hosts to the cluster by using the BMO, you must set the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false`.
+Provisioning bare-metal hosts to the cluster using the BMO sets the `spec.externallyProvisioned` specification in the `BareMetalHost` custom resource to `false` by default. Do not set the `spec.externallyProvisioned` specification to `true`, because this setting results in unexpected behavior.
```

Comment:
I agree with the "by using" here. 

"Provisioning bare-metal hosts to the cluster using the BMO" means "the cluster that is using the BMO" as opposed to the cluster that is not.